rotkehlchen/tasks/calendar.py

```diff
@@ -47,3 +70,96 @@ def delete_past_calendar_entries(database: DBHandler) -> None:
             'DELETE FROM calendar WHERE timestamp < ? AND auto_delete=1',
             (ts_now(),),
         )
+
+
+def maybe_create_ens_reminders(database: DBHandler) -> None:
+    """Check ENS registration and renewal history events and create reminders if needed."""
+    db_calendar = DBCalendar(database=database)
+    db_history_events = DBHistoryEvents(database=database)
+    customizable_date = CustomizableDateMixin(database=database)
```

Comment:
can you maybe initialize the things that you need only if there are events later? if there are no events you will create things that won't be used